virttest/utils_test/libvirt.py

```diff
@@ -2371,19 +2372,46 @@ def create_tpm_dev(params):
     return tpm_dev
 
 
-def get_vm_device(vmxml, dev_tag, index=0):
+def get_vm_device(vmxml, dev_tag, dev_dict={}, index=0):
     """
-    Get current vm device according to device tag
+    Get current vm device according to device tag,
+    if no this device , add one device
```

Comment:
Could you help add a description about the fact that this function only supports to add a new device whose class __init__() has default parameter values.

Reply:
@dzhengfy Thank you dzheng,   I have updated it ,Is this OK

 ```
 if no this device, add a new device and only support the device class
     default parameter values.
```